### PR TITLE
Bump Solution Explorer operator to v0.0.58

### PR DESCRIPTION
**Pull request by sergioifg94**
# Description

* Add new manifests to 0.0.58
* Update currentCSV in channel
* Update `OperatorVersionSolutionExplorer` constant  to "0.0.58"

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: manifests/integreatly-solution-explorer/0.0.58/solution-explorer-operator.v0.0.58.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  maturity: alpha
+  provider: {}
+  version: 0.0.57
```
Comment (pb82) [inline manifests/integreatly-solution-explorer/0.0.58/solution-explorer-operator.v0.0.58.clusterserviceversion.yaml:144]: shouldn't this be 0.0.58 and the same for the next line?

**Comment by mikenairn** [comment]
/assign @philbrookes 

You want to approve this, i have no idea what this change is or if it's intended for the next release.

**Pull request merged**